curriculum/challenges/english/blocks/workshop-bookstore-page/68ec99e478211dc578699944.md

```diff
@@ -0,0 +1,60 @@
+---
+id: 68ec99e478211dc578699944
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Below the `<h2>` element in the first card, add a `<p>` element with this text:
+`This is an epic story of Sally and her dog Rex as they navigate through other worlds.`
```

Comment:
````suggestion
Below the `h2` element in the first `.card` element, add a `p` element with the following text:

```md
This is an epic story of Sally and her dog Rex as they navigate through other worlds.
```
````